packages/cdk/lambda/repository.ts

```diff
@@ -26,6 +26,7 @@ import { getTenantId } from './utils/tenantUtils';
 import { createTenantDynamoDBClient } from './utils/tenantDynamoDBClient';
 
 const TABLE_PREFIX: string = process.env.TABLE_NAME!;
+const ENVIRONMENT: string = process.env.ENVIRONMENT || 'dev';
```

Comment:
特に理由がなければ他の環境変数と同様で良い
```suggestion
const ENVIRONMENT: string = process.env.ENVIRONMENT!;
```